docker-compose.yml

```diff
@@ -41,16 +41,16 @@ services:
       - AGENT_JOBSTATUS=${AGENT_JOBSTATUS}
       - GIT_BRANCH=${GIT_BRANCH}
 
-  bg-geocode:
+  bg:
     build:
       args:
         - DOCKER_REPOSITORY=${dockerHubUsername:-dfedigital}
         - TEACHER_TRAINING_API=${dockerHubImageName:-teacher-training-api}
         - CODE_VERSION=${GIT_BRANCH:-latest}
-      context: bg-geocode
+      context: .
       cache_from:
-        - ${dockerHubUsername:-dfedigital}/teacher-training-bg-geocode:${GIT_BRANCH:-latest}
-    image: ${dockerHubUsername:-dfedigital}/teacher-training-bg-geocode:${GIT_BRANCH:-latest}
+        - ${dockerHubUsername:-dfedigital}/teacher-training-api:${GIT_BRANCH:-latest}
+    image: ${dockerHubUsername:-dfedigital}/teacher-training-api:${GIT_BRANCH:-latest}
```

Comment:
This now uses the generic image, so the command should be overriden here, shoudn't it?

Reply:
done...